detekt-formatting/src/test/kotlin/io/gitlab/arturbosch/detekt/formatting/FormattingRuleSpec.kt

```diff
@@ -85,11 +84,8 @@ class FormattingRuleSpec {
         val expectedPath = Path("src/test/resources/configTests/chain-wrapping-before.kt").toAbsolutePath()
 
         val rule = ChainWrapping(Config.empty)
-        rule.visitFile(compileForTest(expectedPath))
-        assertThat(rule.findings).isNotNull()
-
-        val findings = rule.findings
-
-        assertJThat(findings.first().location.filePath.absolutePath.toString()).isEqualTo(expectedPath.toString())
+        val findings = rule.visitFile(compileForTest(expectedPath))
+        assertThat(findings).isNotNull()
+        assertThat(findings.first().location.filePath.absolutePath.toString()).isEqualTo(expectedPath.toString())
```

Comment:
```suggestion
        assertThat(findings).anySatisfy { finding ->
            assertThat(finding.location.filePath.absolutePath.toString()).isEqualTo(expectedPath.toString())
        }
```
There is no need to check for findings to be `null`.